hooks/lib/module/module.py

```diff
@@ -39,26 +40,4 @@ def get_https_mode(module_name: str, values: dict) -> str:
 
 
 def get_module_name() -> str:
-    module = ""
-    file_path = os.path.abspath(__file__)
-    external_modules_dir = os.getenv("EXTERNAL_MODULES_DIR")
-    for dir in os.getenv("MODULES_DIR").split(":"):
-        if dir.startswith(external_modules_dir):
-            dir = external_modules_dir
-        if file_path.startswith(dir):
-            module = re.sub(f"{dir}/?\d?\d?\d?-?", "",
-                            file_path, 1).split("/")[0]
-            # /deckhouse/external-modules/virtualization/mr/hooks/hook_name.py
-            # {-------------------------- file_path --------------------------}
-            # {------ MODULES_DIR ------}{---------- regexp result ----------}}
-            #                             virtualization/mr/hooks/hook_name.py
-            #                            {-module-name-}{---------------------}
-            #                                  or
-            # /deckhouse/modules/900-virtualization/hooks/hook_name.py
-            # {---------------------- file_path ----------------------}
-            # {-- MODULES_DIR --}{---{-------- regexp result --------}}
-            #                        virtualization/hooks/hook_name.py
-            #                        {-module-name-}{-----------------}
-
-            break
-    return module
+    return MODULE_NAME
```

Comment:
I think, that we must always pass the module name as an argument. This feature should just be removed

Reply:
should delete:
1)
https://github.com/deckhouse/virtualization/blob/683bb70318210e89755de237d813a9720911395d/hooks/lib/hooks/hook.py#L25
2)
https://github.com/deckhouse/virtualization/blob/683bb70318210e89755de237d813a9720911395d/hooks/lib/hooks/hook.py#L43